README.md

```diff
@@ -3,6 +3,14 @@ Getting started on your own!!
 2. cd public-website
 3. npm install
 4. npm start (this creates a new local server running the website, and will open a new browser window)
+5. Remember to run [npm run format](#npm-run-format) before you a PR up for review
+
+6. There are also some environment files you must create (and should never be committed to git):
+  - Google Calendar API:
+    - make a copy of the .env_example file in the root directory public-website and name the copy .env (this is in the .gitignore and should not be committed to git)
+    - get the Calendar ID from google calendar and set the variable REACT_APP_GOOGLE_CALENDAR_ID to it (if you don't have access to the [SE Events](https://calendar.google.com/calendar?cid=bGI2dHRiMWtzMzdxbDVyOGlzdWFzZ2NkbzhAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) calendar contact @RobbieZhuang, @jenniferwills or @yingninggui, and we can help) 
```

Comment:
nit: You could also add in a line about using one's own test calendars for dev work

Reply:
That would simply be replace the Calendar ID to their own calendar right?